test/parser_select_tests.erl

```diff
@@ -567,3 +567,12 @@ single_line_comment_in_multiline_ctrl_select_test() ->
 %             "WHERE a = 'val'"))
 %     ).
 
+select_with_arithmetic_in_where_clause_test() ->
+    Query_sql =
+        "SELECT * FROM mytab "
+        "WHERE a = 10 + 1",
+    {select, Parsed_query} = riak_ql_parser:ql_parse(riak_ql_lexer:get_tokens(Query_sql)),
+    ?assertEqual(
+        {where, [{'=', <<"a">>, {'+',{'integer', 10},{'integer',1}}}]},
+        proplists:lookup(where, Parsed_query)
+    ).
```

Comment:
This test also clearly (w/ some brain power and reading) shows that riak_kv is performing the functionality of resolving literal-op-literal arithmetic before issuing the query to the cluster.